examples/imageload_png_simpletest.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2022 Tim Cocks
+# SPDX-License-Identifier: MIT
+
+import board
+import displayio
+from vectorio import Rectangle
+import adafruit_imageload
+
+# built-in display
+display = board.DISPLAY
+
+# load png image
+image, palette = adafruit_imageload.load("images/test_image.png")
+
+# Set the transparency index color to be hidden
+palette.make_transparent(0)
```

Comment:
This is another todo – we could read the transparency information from the file, there are two kinds of chunks that can define that.